### PR TITLE
LRSD-5033 [Learn] Change side nav template to fit new json structure and move css to globalcss

### PR DESCRIPTION
**Pull request by lluk4z**

### CODE REVIEW

**Comment by lluk4z** [comment]
ci:test:sf

File: workspaces/liferay-learn-workspace/client-extensions/liferay-learn-global-css/assets/global.css [hunk 1]
```diff
@@ -783,6 +785,187 @@ pre[class*='language-'] {
...
+		font-family: 'Source Sans 3';
+		font-size:16px;
+		font-weight: 600;
+		padding: 0.75rem 0rem!important;
+	}
+	  
+	.panel-body .container-lesson:hover{
+		background-color: var(--color-action-primary-hover-10 ,#EDF3FE);
+
+		a{
```
Comment (rtienpal) [inline workspaces/liferay-learn-workspace/client-extensions/liferay-learn-global-css/assets/global.css:938]: give space between a and {

File: workspaces/liferay-learn-workspace/client-extensions/liferay-learn-global-css/assets/global.css [hunk 2]
```diff
@@ -783,6 +785,187 @@ pre[class*='language-'] {
...
+				&:hover {
+					background-color: var(
+						--color-action-primary-hover-10,
+						#edf3fe
+					);
+					color: var(--color-action-primary-hover, #0053f0);
+				}
+
+				&.selected {
+					background-color: var(
```
Comment (rtienpal) [inline workspaces/liferay-learn-workspace/client-extensions/liferay-learn-global-css/assets/global.css:899]: make single line?

**Comment by lluk4z** [comment]
ci:test:sf

**Comment by lluk4z** [comment]
ci:test:sf

**Comment by lluk4z** [comment]
ci:test:sf

**Comment by lluk4z** [comment]
ci:test:sf

**Comment by lluk4z** [comment]
ci:test:sf

**Comment by joseabelenda** [comment]
resent:https://github.com/brianchandotcom/liferay-portal/pull/152065

**Pull request closed without merging**